vortexasdk/endpoints/voyages_congestion_breakdown.py

```diff
@@ -210,6 +214,12 @@ def search(
             order_direction: Determines the direction of sorting. ‘asc’ for ascending, ‘desc’ for
             descending.
 
+            voyage_date_range_activity: Filter to determinate the mode into how the voyages should be counted,`active`, `departures` or `arrivals`.
+
+            origin_behaviour: The origin behaviour into how the voyage_date_range_activity should consider in departure mode, it could be `first_load` or `any_load`.
```

Comment:
This is hard to understand, I think it needs to be rephrased:

```origin_behaviour: The origin behaviour determines which departure mode the `voyage_date_range_activity` should count, must be one of  [`first_load`, `any_load`].```

Reply:
changed too